src/asm/asmy.y

```diff
@@ -142,8 +142,9 @@ void	copyrept( void )
 {
 	SLONG	level=1, len, instring=0;
 	char	*src=pCurrentBuffer->pBuffer;
+	char	*bufferEnd = pCurrentBuffer->pBufferStart + pCurrentBuffer->nBufferSize;
```

Comment:
Why `pCurrentBuffer->pBuffer` for src but `pCurrentBuffer->pBufferStart` for end?